lib/windowsbasetest.pm

```diff
@@ -141,38 +142,31 @@ sub wait_boot_windows {
+        assert_screen(['windows-edge-welcome', 'windows-desktop', 'windows-edge-decline', 'networks-popup-be-discoverable', 'windows-start-menu', 'windows-qemu-drivers', 'windows-setup-browser'], timeout => 120);
         while (not match_has_tag('windows-desktop')) {
+            assert_and_click 'windows-edge-welcome' if (match_has_tag 'windows-edge-welcome');
+            assert_and_click 'windows-setup-browser' if (match_has_tag 'windows-setup-browser');
             assert_and_click 'network-discover-yes' if (match_has_tag 'networks-popup-be-discoverable');
             assert_and_click 'windows-edge-decline' if (match_has_tag 'windows-edge-decline');
             assert_and_click 'windows-start-menu' if (match_has_tag 'windows-start-menu');
             assert_and_click 'windows-qemu-drivers' if (match_has_tag 'windows-qemu-drivers');
             wait_still_screen stilltime => 15, timeout => 60;
-            assert_screen(['windows-desktop', 'windows-edge-decline', 'networks-popup-be-discoverable', 'windows-start-menu', 'windows-qemu-drivers'], timeout => 120);
+            assert_screen(['windows-edge-welcome', 'windows-desktop', 'windows-edge-decline', 'networks-popup-be-discoverable', 'windows-start-menu', 'windows-qemu-drivers', 'windows-setup-browser'], timeout => 120);
         }
 
-        # TODO: all this part should be added to the unattend XML in the future
-
-        # Setup stable lock screen background
-        record_info('Config lockscreen', 'Setup stable lock screen background');
-        $self->use_search_feature('lock screen settings');
-        assert_screen 'windows-lock-screen-in-search';
-        wait_still_screen stilltime => 2, timeout => 10, similarity_level => 43;
-        assert_and_click 'windows-lock-screen-in-search', dclick => 1;
-        assert_screen 'windows-lock-screen-settings';
-        assert_and_click 'windows-lock-screen-background';
-        assert_and_click 'windows-select-picture';
-        assert_and_click 'windows-close-lockscreen';
-        wait_still_screen stilltime => 2, timeout => 10, similarity_level => 43;
+        $self->set_stable_lockscreen unless check_var('VERSION', '11');
 
         # These commands disable notifications that Windows shows randomly and
         # make our windows lose focus
         $self->open_powershell_as_admin;
         $self->run_in_powershell(cmd => 'reg add "HKLM\Software\Policies\Microsoft\Windows" /v Explorer');
         $self->run_in_powershell(cmd => 'reg add "HKLM\Software\Policies\Microsoft\Windows\Explorer" /v DisableNotificationCenter /t REG_DWORD /d 1');
         $self->run_in_powershell(cmd => 'reg add "HKLM\Software\Microsoft\Windows\CurrentVersion\PushNotifications" /v ToastEnabled /t REG_DWORD /d 0');
-        # Prevent password from expiring:
-        $self->run_in_powershell(cmd => 'wmic UserAccount set PasswordExpires=False');
+        # wmic commands are deprecated for 24H2. By default the password expires
```

Comment:
I would just remove this whole block. If this is deprecated now for good, there is no chance we would ever need this again, no? :thinking: